src/pheromones/hivemind-host/src/IpDiscoveryDTO.cpp

```diff
@@ -0,0 +1,30 @@
+#include "IpDiscoveryDTO.h"
+
+IPDiscoveryDTO::IPDiscoveryDTO(const IPDiscovery & ipDiscovery) {
+    m_ip = ipDiscovery.ip;
```

Comment:
You could write

IPDiscoveryDTO::IPDiscoveryDTO(const IPDiscovery & ipDiscovery) : m_ip(ipDiscovery.ip){}